engine/swarm/ingress.md

```diff
@@ -244,7 +244,7 @@ single virtual IP. Instead, Docker sets up DNS entries for the service such that
 a DNS query for the service name returns a list of IP addresses, and the client
 connects directly to one of these. You are responsible for providing the list of
 IP addresses and ports to your load balancer. See
-[Configure service discovery](networking.md@configure-service-discovery).
```

Comment:
Leave it as is but change the `@` to `#` please. :) John has some magic script that does the right thing with the `.md` suffixes.

Reply:
@mistyhacks ... are you saying, remove the `/`? Because that's the only difference...

Reply:
The problem is that the `@` is supposed to be a `#`...  so change

```
-[Configure service discovery](networking.md@configure-service-discovery).
```

to

```
-[Configure service discovery](networking.md#configure-service-discovery).
```